common/lib/xmodule/xmodule/lti_module.py

```diff
@@ -584,7 +585,8 @@ def get_resource_link_id(self):
         i4x-2-3-lti-31de800015cf4afb973356dbe81496df this part of resource_link_id:
         makes resource_link_id to be unique among courses inside same system.
         """
-        return str(parse.quote(f"{self.system.hostname}-{self.location.html_id()}"))  # lint-amnesty, pylint: disable=line-too-long
+        hostname = getattr(settings, "LMS_BASE", "")
```

Comment:
This is already defined in `envs/common.py`, so I believe we don't need a fallback. We should be able to simply use `settings.LMS_BASE`. The same applies to the backward-compatible `hostname` property in `common/lib/xmodule/xmodule/x_module.py`.

Reply:
Update in both cases.